### ISSUE TITLE
[Bug] Bug in Baldwin when restricting to 2 candidates

### ISSUE THREAD

**Issue by DominikPeters** [voting-tools/pref_voting#89]
The following code produces an error:
```python
from pref_voting.profiles import Profile
from pref_voting.iterative_methods import baldwin

prof = Profile([[0, 1, 2, 3]])
baldwin(prof, curr_cands=[0, 1])
```
namely
```
ValueError                                
Traceback (most recent call last)
Cell In[6], line 5
      2 from pref_voting.iterative_methods import baldwin
      4 prof = Profile([[0, 1, 2, 3]])
----> 5 baldwin(prof, curr_cands=[0, 1])

File pref_voting/voting_method.py:34, in VotingMethod.__call__(self, edata, curr_cands, **kwargs)
     32 if (curr_cands is not None and len(curr_cands) == 0) or len(edata.candidates) == 0: 
     33     return []
---> 34 return self.vm(edata, curr_cands = curr_cands, **kwargs)

File pref_voting/iterative_methods.py:925, in baldwin(profile, curr_cands)
    923 else: # remove the candidates with lowest Borda score
    924     num_cands = len(candidates)
--> 925     updated_rankings = _find_updated_profile(rankings, cands_to_ignore, num_cands)
    927 while len(winners) == 0:
    928     borda_scores = {c: _borda_score(updated_rankings, rcounts, num_cands - cands_to_ignore.shape[0], c) for c in candidates if not isin(cands_to_ignore, c)}

ValueError: negative dimensions not allowed
```

**Comment by epacuit** [comment]
Thank! We'll  look into this.

**Comment by epacuit** [comment]
This bug should now  be fixed in the latest version (1.1.2).    Thanks again!

**Comment by DominikPeters** [comment]
Thanks, great!
